internal/config/config.go

```diff
@@ -162,6 +167,8 @@ func parseParameters(secretStr, parametersStr string, defaultAkeylessGatewayURL
 	parameters.AkeylessAzureObjectID = params["akeylessAzureObjectID"]
 	parameters.AkeylessGCPAudience = params["akeylessGCPAudience"]
 	parameters.AkeylessUIDInitToken = params["akeylessUIDInitToken"]
+	parameters.AkeylessK8sServiceAccountToken = params["akeylessK8sServiceAccountToken"]
```

Comment:
please remove AkeylessK8sServiceAccountToken
no use in the code

Reply:
fixed